admin/acceptance_tests/layertree_test.py

```diff
@@ -142,13 +142,13 @@ def test_unlink_view(self, test_app, layertree_test_data, dbsession):
     @pytest.mark.usefixtures("selenium", "selenium_app")
     def test_unlink_selenium(self, dbsession, selenium, selenium_app, layertree_test_data):
         from selenium.webdriver.common.by import By
-        from selenium.webdriver.support import expected_conditions as EC
+        from selenium.webdriver.support import expected_conditions
```

Comment:
Note that EC comes directly from selenium python documentation : http://selenium-python.readthedocs.io/waits.html#explicit-waits

Reply:
But it don't pass the code style, it's the only reason of this change...